cellxgene_schema_cli/requirements.txt

```diff
@@ -1,12 +1,12 @@
-anndata==0.8.0
-cellxgene-ontology-guide==0.4.0
-click==8.1.3
-Cython==0.29.34
-numpy==1.23.2
-pandas==1.4.4
-PyYaml==6.0
+anndata==0.10.7
+cellxgene-ontology-guide==0.4.0 # update before a schema migration
+click<9
+Cython<4
+numpy<2
+pandas<2
+PyYAML<7
 wheel==0.40.0
-scipy<1.13.0    # scipy 1.13.0 is not compatible with anndata <=0.10.6, revisit before next release
-semver==3.0.0
-xxhash==3.3.0
-matplotlib==3.7.3
+scipy==1.13.0    # scipy 1.13.0 is not compatible with anndata <=0.10.6, revisit before next release
+semver<4
+xxhash<4
+matplotlib<4
```

Comment:
the schema reference provides a definition for supported "colors" to use as values for "colors" keys in the uns. We should double check that having that list is compatible with updating this library. Or maybe include which version of matplotlib we use to validate in the schema reference.

Reply:
looking at the matplotlib change logs, there have been no changes to colors between 3.7.3 and 3.9.0. The portal has all the [named colors](https://github.com/chanzuckerberg/single-cell-data-portal/blob/6357aa21d5a5da3986310f1dd595769f0e03422e/backend/common/utils/color_conversion_utils.py) hardcoded and those also haven't change. I think we should remove this as a requirement from the CLI and hardcode `matplotlib.colors` somewhere in the code.